src/me/components/AlertsStatuses.tsx

```diff
@@ -0,0 +1,89 @@
+// Libraries
+import React, {FC, useMemo} from 'react'
+
+// Components
+import {Panel, Heading, HeadingElement, FontWeight} from '@influxdata/clockface'
+import {
+  runAlertsStatusesQuery,
+  STATUSES_FIELDS,
+} from 'src/alerting/utils/statuses'
+import {connect} from 'react-redux'
+import EventViewer from 'src/eventViewer/components/EventViewer'
+import EventTable from 'src/eventViewer/components/EventTable'
+import {ResourceType} from 'src/types'
+import {ResourceIDsContext} from 'src/alerting/components/AlertHistoryIndex'
+import {ResourceIDs} from 'src/checks/reducers'
+import {RouteComponentProps, useParams} from 'react-router'
+import GetResources from 'src/resources/components/GetResources'
+import {getCheckIDs} from 'src/checks/selectors'
+import {AppState} from 'src/types'
+
+const fields = STATUSES_FIELDS
+
+interface StateProps {
+  resourceIDs: ResourceIDs
+}
+
+type Props = RouteComponentProps<{orgID: string}> & StateProps
+
+const AlertsStatuses: FC<Omit<Props, 'history' | 'location' | 'match'>> = ({
+  resourceIDs,
+}) => {
+  const {orgID} = useParams<{orgID: string}>()
+
+  const loadRows = useMemo(
+    () => options => runAlertsStatusesQuery(orgID, options),
+    [orgID]
+  )
+
+  return (
+    <GetResources resources={[ResourceType.Checks]}>
+      <ResourceIDsContext.Provider value={resourceIDs}>
+        <Panel>
+          <Panel.Header>
+            <Heading
+              element={HeadingElement.H2}
+              weight={FontWeight.Light}
+              className="cf-heading__h4"
+              testID="alerts-statuses"
+            >
+              Alerts Statuses
```

Comment:
Is "Alerts Activity" better? "Alerts Statuses" seems kinda weird to me English language-wise. 
Maybe subhead that says "Health Report for Checks, Notification Rules, Endpoints" <-- perhaps that's too much?
Thoughts @juliajames12?

Reply:
haha. yeah. it does sound weird. I had the other option "Alerts Statii" 🤣 
let me know what needs to be final.

Reply:
My vote is "Alerts Activity"